app/models/container_template.rb

```diff
@@ -0,0 +1,55 @@
+class ContainerTemplate < ApplicationRecord
+  include CustomAttributeMixin
+
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :container_project
+  has_many :container_template_parameters, :dependent => :destroy
+  has_many :labels, -> { where(:section => "labels") },
+           :class_name => CustomAttribute,
+           :as         => :resource,
+           :dependent  => :destroy
+
+  serialize :objects, Array
+
+  acts_as_miq_taggable
+
+  def instantiate(params)
+    temp = {}
+    temp[:metadata] = {}
+    temp[:metadata][:name] = name
+    temp[:metadata][:namespace] = container_project.name
+    temp[:objects] = objects
+    temp[:parameters] = params
```

Comment:
we would like to allow the user to edit the template parameters and not just use them as they are (to be added later on)